client/components/home/Pickup/carousel.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Flex, Stack, HStack, Image } from "@chakra-ui/react";
+import { useState, useEffect } from "react";
+import { COLORS } from "theme";
+import { useBreakpointValue } from "@chakra-ui/react";
+
+const PickupCarousel = () => {
+	const slides = [
+		{
+			url: useBreakpointValue({
+				base: "/headerBanner1mobile.png",
+				md: "/headerBanner1desktop.png",
+			}),
+			number: 20,
+		},
+		{
+			url: useBreakpointValue({
+				base: "/headerBanner2mobile.png",
+				md: "/headerBanner2desktop.png",
+			}),
+			distance: "20m",
+		},
+	];
+	const [currentSlide, setCurrentSlide] = useState(0);
+	const slidesCount = slides.length;
+	const SLIDES_INTERVAL_TIME = 20000;
+	const ANIMATION_DIRECTION = "right";
+	const carouselStyle = {
+		transition: "all .5s",
+		ml: `-${currentSlide * 100}%`,
+	};
+	useEffect(() => {
+		const prevSlide = () => {
+			setCurrentSlide((s) => (s === 0 ? slidesCount - 1 : s - 1));
+		};
+
+		const nextSlide = () => {
+			setCurrentSlide((s) => (s === slidesCount - 1 ? 0 : s + 1));
+		};
+
+		const automatedSlide = setInterval(() => {
+			ANIMATION_DIRECTION.toLowerCase() === "left" ? prevSlide() : nextSlide();
```

Comment:
will the ANIMATION_DIRECTION ever change as it's initialised as a const

if no, then no need for this logic

Reply:
removed the logic since we only have 2 cards. also removed prevSlide. verified that the carousel still works